libs/sr/ui/src/LightCone/LightConeCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { CardThemed } from '@genshin-optimizer/common/ui'
+import { convert, selfTag } from '@genshin-optimizer/sr/formula'
+import { CardContent, Typography } from '@mui/material'
+import { useCalcContext } from '../Context'
+import { useLightCone } from '../Hook'
+
+type LightConeCardProps = {
+  lightConeId: string
+}
+export function LightConeCard({ lightConeId }: LightConeCardProps) {
+  const lightCone = useLightCone(lightConeId)
+  const { calc } = useCalcContext()
```

Comment:
```suggestion
  const calc = srCalculatorWithEntries(
    withMember('member0', ...lightConeData(lightCone))
  )
```

Add relevant imports. This should fix the stat calculation